Lib/test/test_embed.py

```diff
@@ -1782,10 +1782,8 @@ def test_initconfig_api(self):
             'perf_profiling': 2,
         }
         config_dev_mode(preconfig, config)
-        # Temporarily enable ignore_stderr=True to ignore warnings on JIT builds
-        # See gh-126255 for more information
         self.check_all_configs("test_initconfig_api", config, preconfig,
-                               api=API_ISOLATED, ignore_stderr=True)
```

Comment:
That's fun. Due to incorrect assumptions, where JIT is actually enabled or not, this hack is no longer needed. Just setting PYTHON_JIT=0 is enough here.